lib/db/schema/patch-067-068.sql

```diff
@@ -0,0 +1,165 @@
+ALTER TABLE `unverifiedTokens`
+ADD COLUMN `tokenVerificationCodeHash` BINARY(32) DEFAULT NULL,
+ADD COLUMN `tokenVerificationCodeExpiresAt` BIGINT UNSIGNED DEFAULT NULL,
+ALGORITHM = INPLACE, LOCK = NONE;
+
+CREATE INDEX `unverifiedTokens_tokenVerificationCodeHash_uid`
+ON `unverifiedTokens` (`tokenVerificationCodeHash`, `uid`)
+ALGORITHM = INPLACE LOCK=NONE;
+
+CREATE PROCEDURE `verifyTokenCode_1` (
+  IN `tokenVerificationCodeHashArg` BINARY(32),
+  IN `uidArg` BINARY(16)
+)
+BEGIN
+  DECLARE EXIT HANDLER FOR SQLEXCEPTION
+  BEGIN
+    ROLLBACK;
+    RESIGNAL;
+  END;
+
+  SET @tokenVerificationId = NULL;
+  SELECT tokenVerificationId INTO @tokenVerificationId FROM unverifiedTokens
+    WHERE uid = uidArg
+    AND tokenVerificationCodeHash = tokenVerificationCodeHashArg
+    AND tokenVerificationCodeExpiresAt >= (UNIX_TIMESTAMP(NOW(3)) * 1000);
+
+  IF @tokenVerificationId IS NULL THEN
+    SET @updateCount = 0;
```

Comment:
tiny nit: I don't really understand what the name "updateCount" is supposed to mean here; maybe "expiredCount" or something to make it clearer?